src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,121 @@
+
+For example, when a potential victim is using a service to help them leave a domestic abuser.
+
+You can use this component on either:
+- all the pages in a service
+- parts of the journey with sensitive information
+
+You can also use this component for standalone content pages, such as dashboards and guidance.
+
+## When not to use this component
+
+Do not use this component if the service or content is unlikely to put a user at risk. See the [Exit a page quickly](/patterns/exit-a-page-quickly/) pattern for examples of at-risk and sensitive topics.
+
+The 'Exit this page' component is a [button component](/components/button/) that has been marked up with addtional CSS and JavaScript functionality, to make it work in a specific way. 
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use this component. You can use HTML or, if you're using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+### Adding the button
+
+Position the component at the top of your page, above the beginning of the grid (`<div class="govuk-grid-row">`) but still within the width container (`<div class="govuk-width-container">`).
+
+### Choosing a different website to load
+
+When activated, the component by default will take the user to the [homepage of BBC Weather](https://www.bbc.co.uk/weather). You can change this if there’s a more appropriate site for your users.
+
+Avoid websites that might show personalised pages (such as frequently visited, last visited or suggested links), as this content could put the user at risk.
+
+## Consider what to do with user session data
+
+You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.
+
+For example, you could do this in 2 steps:
+1. replace the default website link to a URL in your service, that when visited, will clear the user's session data
+2. redirect the user to another website, such as BBC Weather
+
+## Adding the secondary link
+
+Add the code for the 'secondary link' into the layout file of your service. The 'secondary link' is an additional `govuk-skip-link` that works in a similar way. Place it under the page template's default skip link at the very top of the body.
+
+When the user interacts with the 'secondary link', it will perform the same action as pressing the button to activate Exit this Page.
+
+{{ example({group: "components", item: "exit-this-page", example: "secondary-link", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+The `href` for the 'secondary link' should be the same as the URL used for the button.
+
+The component will still work if the 'secondary link' is not added, but it'll be less accessible overall.
```

Comment:
```suggestion
## Adding the secondary link

Add the code for the 'secondary link' into the layout file of your service. The 'secondary link' is an additional `govuk-skip-link` that works in a similar way. Place it under the page template's default skip link at the very top of the body.

When the user interacts with the 'secondary link', it will perform the same action as pressing the button to activate Exit this Page.

{{ example({group: "components", item: "exit-this-page", example: "secondary-link", html: true, nunjucks: true, open: false, size: "s"}) }}

The `href` for the 'secondary link' should be the same as the URL used for the button.

The component will still work if the 'secondary link' is not added, but it'll be less accessible overall.

## Consider what to do with user session data

You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.

For example, you could do this in 2 steps:
1. replace the default website link to a URL in your service, that when visited, will clear the user's session data
2. redirect the user to another website, such as BBC Weather

```